pkg/gadgets/trace/open/tracer/tracer_test.go

```diff
@@ -174,13 +197,97 @@ func TestOpenTracer(t *testing.T) {
 					Ret:           fd,
 					Err:           0,
 					Path:          "/tmp/test",
+					FullPath:      "/tmp/test",
 					Flags:         []string{"O_RDWR", "O_CREAT"},
 					FlagsRaw:      unix.O_CREAT | unix.O_RDWR,
 					Mode:          "-rwxrw---x",
 					ModeRaw:       unix.S_IRWXU | unix.S_IRGRP | unix.S_IWGRP | unix.S_IXOTH,
 				}
 			}),
 		},
+		"test_relative_path": {
+			getTracerConfig: func(info *utilstest.RunnerInfo) *tracer.Config {
+				return &tracer.Config{
+					MountnsMap: utilstest.CreateMntNsFilterMap(t, info.MountNsID),
+				}
+			},
+			generateEvent: func() (int, error) {
+				relPath := generateRelativePathForAbsolutePath("/tmp/test")
+				fd, err := unix.Open(relPath, unix.O_CREAT|unix.O_RDWR, unix.S_IRWXU|unix.S_IRGRP|unix.S_IWGRP|unix.S_IXOTH)
+				if err != nil {
+					return 0, fmt.Errorf("opening file: %w", err)
+				}
+
+				defer os.Remove(relPath)
+
+				unix.Close(fd)
+
+				return fd, nil
+			},
+			validateEvent: utilstest.ExpectOneEvent(func(info *utilstest.RunnerInfo, fd int) *types.Event {
+				relPath := generateRelativePathForAbsolutePath("/tmp/test")
+				return &types.Event{
+					Event: eventtypes.Event{
+						Type: eventtypes.NORMAL,
+					},
+					WithMountNsID: eventtypes.WithMountNsID{MountNsID: info.MountNsID},
+					Pid:           uint32(info.Pid),
+					Uid:           uint32(info.Uid),
+					Comm:          info.Comm,
+					Fd:            fd,
+					Ret:           fd,
+					Err:           0,
+					Path:          relPath,
+					FullPath:      "/tmp/test",
+					Flags:         []string{"O_RDWR", "O_CREAT"},
+					FlagsRaw:      unix.O_CREAT | unix.O_RDWR,
+					Mode:          "-rwxrw---x",
+					ModeRaw:       unix.S_IRWXU | unix.S_IRGRP | unix.S_IWGRP | unix.S_IXOTH,
+				}
+			}),
+		},
+		"test_symbolic_links": {
+			getTracerConfig: func(info *utilstest.RunnerInfo) *tracer.Config {
+				return &tracer.Config{
+					MountnsMap: utilstest.CreateMntNsFilterMap(t, info.MountNsID),
+				}
+			},
+			generateEvent: func() (int, error) {
+				// Create a symbolic link to /dev/null
+				err := os.Symlink("/dev/null", "/tmp/test")
```

Comment:
Tests are executed in parallel (`t.Parallel()` line 364) and the path `/tmp/test` is reused by several tests to create a file, create a symlink, or delete it. We need to use different paths in different tests to avoid race conditions.

This problem was not introduced in this PR, but I just noticed it.